src/openrct2/world/Sprite.h

```diff
@@ -61,15 +61,18 @@ struct Duck : SpriteGeneric
     int16_t target_y;
     uint8_t state;
 
+    void Update();
+    uint32_t GetFrameImage(int32_t direction) const;
+    void Invalidate();
+    bool IsFlying();
+    void Remove();
```

Comment:
Why unprotecting all of these in this changeset?

Reply:
They were always unprotected as it's a struct. I added protection to the update sub functions though as they no longer have to be public.

Reply:
Ah, I see, I had missed that `private` was one of the additions as well

Reply:
If you're adding private members, this should probably be a class instead.

Reply:
I was adding a public member. It's just the 4 existing public members no longer need to be public.